srtcore/epoll.h

```diff
@@ -301,7 +363,13 @@ friend class CRendezvousQueue;
       /// create a new EPoll.
       /// @return new EPoll ID if success, otherwise an error number.
 
-   int create();
+   int create(CEPollDesc** ppd = 0);
+
+
+   /// delete all user sockets (SRT sockets) from an EPoll
```

Comment:
change comment alignment to match the rest of the code

Reply:
Reformatted the whole class, just with existing 3-space indent (to minimize changes).